minor doc updates and fixes, intro chapter (#4608)
edgedb/edgedb: docs/intro/edgeql.rst

    const result = await query.run(client);
    // {id: string; title: number}[]

> Note that you reference properties of the object being ``select`` ed by

Reviewer: ```suggestion
Note that you reference properties of the object to include in your ``select``
```

> prepending the property name with a period: ``.release_year``. This is known

Reviewer: ```suggestion
by prepending the property name with a period: ``.release_year``. This is known
```

This should only be committed if alongside the previous suggestion.
